docs/terminal/appearance.md

```diff
@@ -0,0 +1,132 @@
+
+Whether bold text uses the normal ANSI colors or the bright varient can be configured with the `terminal.integrated.drawBoldTextInBrightColors` setting.
+
+### Minimum contrast ratio
+
+Terminals often have contrast issues because of how they were designed, these issues often arise due to some conflict with dark/light themes, ANSI colors or shells/programs running in the terminal.
+
+The minimum contrast ratio feature aims to solve this problem in a general way by always displaying text such that a certain contrast ratio is met. This is done by either increasing or reducing the luminance of the text's foreground color. By default this is set to ensure a 4.5:1 contrast ratio is met for reasonable contrast on all text.
+
+One downside of this is that colored text may sometimes lose some of its saturation. This feature can be disabled to get the original colors with:
+
+```json
+"terminal.integrated.minimumContrastRatio": 1
+```
+
+## GPU acceleration
+
+The terminal features 3 different renderers, each of which have different trade offs:
+
+- Webgl renderer - True GPU acceleration
+- Canvas renderer - GPU acceleration by leveraging the [`CanvasRenderingContext2D` web API](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D), slower than webgl but faster than DOM
+- DOM renderer - A fallback renderer that's much slower but has great compatibility
+
+GPU acceleration driven by the Webgl renderer is enabled in the terminal by default. This helps the terminal work faster and display at a high FPS by significantly reducing the time the CPU spend rendering each frame. This typically works without issue but for certain environments it may not work properly, common examples of this are Linux VMs, browsers that don't support webgl and machines with outdated drivers.
+
+The default `terminal.integrated.gpuAcceleration` value of `"auto"` work by first selecting the webgl renderer, if it detects issues it will fallback to the canvas renderer, and if it detect issues there it will fallback to the DOM renderer. Sometimes this detection doesn't work and requires manual intervention, setting `terminal.integrated.gpuAcceleration` to `"dom"` typically resolves rendering-related problems like these at the cost of performance.
+
+### Custom glyphs
+
+When [GPU acceleration](#_gpu-acceleration) is enabled, various special characters use custom rendering instead of the font to improve how they display in the terminal. These include box drawing characters (`U+2500-U+257F`), block elements (`U+2580-U+259F`) and a subset of powerline symbols (`U+E0B0-U+E0B7`). This means that the configured font does not need to support these characters as well as having the characters draw pixel perfect and stretch to the size of the entire cell.
```

Comment:
```suggestion
When [GPU acceleration](#_gpu-acceleration) is enabled, custom rendering, rather than the font, improves how some characters display in the terminal. These include box drawing characters (`U+2500-U+257F`), block elements (`U+2580-U+259F`) and a subset of powerline symbols (`U+E0B0-U+E0B7`). This means that the configured font does not need to support these characters as well as having the characters draw pixel perfect and stretch to the size of the entire cell.
```